frontend/src/Components/UserProfileSettings.tsx

```diff
@@ -67,6 +75,7 @@ export default function UserProfileSettings(props: UserProfileSettingsProps) {
 const [autoStop, setAutoStop] = React.useState<boolean>(getVideoAutopause());
 const [legacyZoom, setLegacyZoom] = React.useState<boolean>(getLegacyZoom());
 const [preferredLang, setPreferredLang] = React.useState<string>(getPreferredLang());
+const [indentScale, setIndentScale] = React.useState<string>(getIndentScale());
```

Comment:
Could be replaced with `useLocalStorage` hook. See example here:
https://github.com/spaceshelter/orbitar/blob/02f60bfdea89b6d5aa19db6886bcf2569f1fa7b9/frontend/src/Components/Topbar.tsx#L74

Reply:
Added default Auto option.